ci/redmine_bugzilla.py

```diff
@@ -149,12 +150,22 @@ def main():
                         links_issues_record += 'Redmine #%s -> Bugzilla %s, but Bugzilla %s does not ' \
                                            'link back\n' % (issue.id, bug.id, bug.id)
 
+    start_time = time.time()
```

Comment:
This 1 start_time is being used for multiple measurements. For the total time this makes sense, but it will be wrong for individual queries on L#162. I suggest having a separate start time for each query in L#162 so that the measurement on L#165 will be only for that 1 query. Using separate, and good variable names would be good.